### ISSUE TITLE
Update to composer 2

### ISSUE THREAD

**Issue by groovytron** [groovytron/php-container#9]
To keep compatibility, we can maybe install 2 composer versions so that we have the following links:

``` console
$ composer --version
Composer version 2.0.8 2020-12-03 17:20:38
$ composer1 --version
Composer version 1.10.19 2020-12-04 09:14:16
```

**Comment by groovytron** [comment]
The use of `prestissimo` is not needed anymore (and it's not compatible with composer 2 anyway).

**Comment by groovytron** [comment]
Implemented in [1.3.0](https://github.com/groovytron/php-container/compare/1.2.1...1.3.0)
